omnigibson/robots/fetch.py

```diff
@@ -388,6 +392,52 @@ def disabled_collision_pairs(self):
             ["torso_lift_link", "shoulder_lift_link"],
             ["torso_lift_link", "torso_fixed_link"],
         ]
+    
+    @property
+    def primitive_disabled_collision_pairs(self):
+        """
+        Returns:
+            Array of arrays: Disabled collision pairs for necessary for primitives. Including these in disabled_collision_pairs
+            throws an error. When bug is fixed, these can be merged with disabled_collision_pairs.
```

Comment:
what bug needs to be fixed? is this needed in 2023.1.0?

Reply:
@cgokmen @cremebrule https://stanfordvl.slack.com/archives/C05DFPGND7G/p1689372142805389

Reply:
Yeah this seems fixed. Merging the two.